internal/thelma/cli/commands/bees/apply_schedule/apply_schedule_command.go

```diff
@@ -0,0 +1,231 @@
+package apply_schedule
+
+import (
+	"fmt"
+	"github.com/broadinstitute/thelma/internal/thelma/app"
+	"github.com/broadinstitute/thelma/internal/thelma/cli"
+	"github.com/broadinstitute/thelma/internal/thelma/cli/commands/bee/common/builders"
+	"github.com/broadinstitute/thelma/internal/thelma/cli/commands/bee/common/filterflags"
+	"github.com/broadinstitute/thelma/internal/thelma/cli/commands/bee/common/views"
+	"github.com/broadinstitute/thelma/internal/thelma/state/api/terra"
+	"github.com/broadinstitute/thelma/internal/thelma/toolbox/argocd"
+	"github.com/broadinstitute/thelma/internal/thelma/utils/pool"
+	"github.com/broadinstitute/thelma/internal/thelma/utils/schedule"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"sync"
+	"time"
+)
+
+const helpMessage = `Start and stop BEEs as defined by their schedule`
+
+type options struct {
+	dryRun      bool
+	start       bool
+	stop        bool
+	fromPast    time.Duration
+	maxParallel int
+}
+
+var flagNames = struct {
+	dryRun      string
+	start       string
+	stop        string
+	fromPast    string
+	maxParallel string
+}{
+	dryRun:      "dry-run",
+	start:       "start",
+	stop:        "stop",
+	fromPast:    "from-past",
+	maxParallel: "max-parallel",
+}
+
+type command struct {
+	options options
+	fflags  filterflags.FilterFlags
+}
+
+func NewBeesApplyScheduleCommand() cli.ThelmaCommand {
+	return &command{
+		fflags: filterflags.NewFilterFlags(),
+	}
+}
+
+func (cmd *command) ConfigureCobra(cobraCommand *cobra.Command) {
+	cobraCommand.Use = "apply-schedule"
+	cobraCommand.Short = helpMessage
+	cobraCommand.Long = helpMessage
+
+	cobraCommand.Flags().BoolVar(&cmd.options.dryRun, flagNames.dryRun, true, "Print the actions that would be taken instead of also doing them")
+	cobraCommand.Flags().BoolVar(&cmd.options.start, flagNames.start, true, "If start schedules should be applied")
+	cobraCommand.Flags().BoolVar(&cmd.options.stop, flagNames.stop, true, "if stop schedules should be applied")
+	cobraCommand.Flags().DurationVar(&cmd.options.fromPast, flagNames.fromPast, 20*time.Minute, "How far back to look for schedule transitions to apply")
```

Comment:
Maybe include an example of what an argument to this command should look like. I'm guessing it's the usual go duration syntax like `5m`, `1h` etc..?

Reply:
Yeah, cobra is actually kind enough to include a tiny bit of that, will add some more because it only documents the default value as 20m0s which is confusing on its own.